### PR TITLE
Add Ryan Johnson as collaborator

### PR DESCRIPTION
**Pull request by appilon**
@tenthirtyam If you would like your title or the the callout to collaborators from VMware sentence tweaked just let us know.

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ The HashiCorp Terraform vSphere provider team is :
 * Alex Pilon, Engineer - [@appilon](https://github.com/appilon)
 * Brandy Jackson, Engineering Manager - [@ibrandyjackson](https://github.com/ibrandyjackson)
 
+Collaborators from VMware are :
+
+* Ryan Johnson - [@tenthirtyam](https://github.com/tenthirtyam)
```
Comment (appilon) [inline docs/FAQ.md:13]: We can include your title if you prefer? Just let us know:
```
* Ryan Johnson, Staff II Solutions Architect - [@tenthirtyam](https://github.com/tenthirtyam)
```

Comment (tenthirtyam) [reply docs/FAQ.md:13]: Yes, please - thank you!

**Review by tenthirtyam — APPROVED** [review]
I'm good either way. My details are on my GH profile.

**Review by tenthirtyam — APPROVED** [review]
🚀 LGTM! TY!

**Pull request merged**